docs/src/pages/components/alert/AlertPropControls.tsx

```diff
@@ -0,0 +1,96 @@
+import {
```

Comment:
Moving components for demo controls to the same directory to clean up the docs/src/components directory so it only contains components for the docs site.

Reply:
Nice, I like this